src/marqo/core/models/marqo_index.py

```diff
@@ -166,6 +184,10 @@ def _update_model_properties_from_registry(self) -> None:
                     f'Could not find model properties for model={model_name}. '
                     f'Please check that the model name is correct. '
                     f'Please provide model_properties if the model is a custom model and is not supported by default')
+            except InvalidModelPropertiesError as e:
+                raise InvalidArgumentError(
+                    f'Invalid model properties for model={model_name}. Reason: {e}.'
```

Comment:
When does this happen?

Reply:
This happens if the user is trying to load a `no_model` without providing `model_properties`.  We will populate the properties from the registry while the model properties in the registry is invalid for `no_model`